include/rdma/fi_domain.h

```diff
@@ -112,6 +112,11 @@ struct fid_mr {
 	uint64_t		key;
 };
 
+enum fi_hmem_iface {
+	FI_HMEM_NONE	= 0,
+	FI_HMEM_CUDA,
+};
+
 struct fi_mr_attr {
```

Comment:
Is the device identifier coming in a separate patch?

Reply:
I think we could add the device identifier here. No need for a separate patch.

Reply:
+1.

Reply:
Updated with device identifier change